### ISSUE TITLE
Scrap Reddit Titles

### ISSUE THREAD

**Issue by ItsNoted** [sissbruecker/linkding#154]
I tried adding a Reddit post as a bookmark and it wasn't able to scrape the title. I don't really care about the content as much as the title.

**Comment by victor-gp** [comment]
(On Firefox) It seems this is an issue with how bookmarks can be saved without an explicit title, then linkding reloads that title every time you access your bookmarks page.

Detailed account:

- I click Add Bookmark in the FF extension, on a reddit page
- the extension picks up the correct page title from the FF tab
- I click save without changing the title cause the placeholder is ok
- the bookmark is stored without title (?)
- when I open my linkding bookmarks, the title for this one is reloaded with an unauthenticated request to Reddit
- to which Reddit responds with a generic page title, "Reddit - Explora lo que quieras" in Spain

On the other hand, if I manually set* a title for the bookmark, the page title seems to stay put.

\* be it in the extension popup or by editing the bookmark in Linkding

Bottom line: I think this can be solved if the extension includes the title placeholder on save?

**Comment by sissbruecker** [comment]
Some improvements to the scraping have been made here: https://github.com/sissbruecker/linkding/pull/263

Another solution would be to move scraping to the client which can be tracked here: #118
